docs/content/examples/success-messages.html

```diff
@@ -0,0 +1,103 @@
+        </thead>
+        <tbody>
+            <tr>
+                <td>“Successfully saved.”</td>
+                <td>“Your account details have been saved.”</td>
+            </tr>
+            <tr>
+                <td>“Upgrade complete.”</td>
+                <td>“You’ve upgraded to the Business tier of Teams.”</td>
+            </tr>
+        </tbody>
+    </table>
+</section>
+<section class="stacks-section">
+    {% header h2 | Use the right tone %}
+    <p class="stacks-copy">
+        <a href="{{ "/content/guidelines/voice-and-tone#tone" | relative_url }}">Tone is how you say something</a>. Excitement makes sense for successfully finishing a large task, but avoid overdoing it for a regular system success. Limit <a href="{{ "/content/guidelines/grammar-and-mechanics#exclamation-points" | relative_url }}">exclamation marks</a> to one per page.
+    </p>
+    <table class="s-table s-table__lg fs-body2 va-top w100 wmx6 mb48">
+        <thead>
+            <th class="btw0 bxw0 bbw3 bbw3 bc-red-2 fc-red-500 bg-transparent fs-body1 s-table--cell4">{% icon Clear | mtn1 mbn1 %} Don’t</th>
+            <th class="btw0 bxw0 bbw3 bc-green-2 fc-green-500 bg-transparent fs-body1 s-table--cell4">{% icon Checkmark | mtn1 mbn1 %} Do</th>
+        </thead>
+        <tbody>
+            <tr>
+                <td>“Thanks for updating your email.”</td>
+                <td>“Your email has been updated.”</td>
+            </tr>
+            <tr>
+                <td>“Account created.”</td>
+                <td>“Thanks for signing up. Your account has been created.”</td>
+            </tr>
+        </tbody>
+    </table>
+</section>
+<section class="stacks-section">
+    {% header h2 | What happens next %}
+    <p class="stacks-copy">
+        Is the success part of a larger goal? Are there other recommended actions that need to be performed? Success messages can be a good way to guide the user to the next action for a more seamless interaction.
+    </p>
+    <table class="s-table s-table__lg fs-body2 va-top w100 wmx6 mb48">
+        <thead>
+            <th class="btw0 bxw0 bbw3 bbw3 bc-red-2 fc-red-500 bg-transparent fs-body1 s-table--cell4">{% icon Clear | mtn1 mbn1 %} Don’t</th>
+            <th class="btw0 bxw0 bbw3 bc-green-2 fc-green-500 bg-transparent fs-body1 s-table--cell4">{% icon Checkmark | mtn1 mbn1 %} Do</th>
+        </thead>
+        <tbody>
+            <tr>
+                <td>“We’ve saved your profile changes.”
+                  <div class="grid gs8 gsx ai-center">
+                      <button class="grid--cell s-btn s-btn__primary js-modal-primary-btn" type="button">OK</button>
+                  </div>
+                </td>
+                <td>“We’ve saved your profile changes.”
+                  <div class="grid gs8 gsx ai-center">
+                      <button class="grid--cell s-btn s-btn__primary js-modal-primary-btn" type="button">View profile</button>
+                  </div>
```

Comment:
`<button class="s-btn s-btn__primary" type="button">View profile</button>`

You can kill the wrapping grid classes on this.